VAMobile/src/screens/HealthScreen/SecureMessaging/SecureMessaging.tsx

```diff
@@ -109,18 +104,12 @@ const SecureMessaging: FC<SecureMessagingScreen> = ({ navigation }) => {
       <StartNewMessageButton />
       <Box flex={1} justifyContent="flex-start">
         <Box mb={theme.dimensions.standardMarginBetween} mt={theme.dimensions.contentMarginTop} mx={theme.dimensions.gutter}>
-          <SegmentedControl
-            values={controlValues}
-            titles={controlLabels}
-            onChange={onTabUpdate}
-            selected={controlValues.indexOf(secureMessagingTab || SecureMessagingTabTypesConstants.INBOX)}
-            accessibilityHints={a11yHints}
-          />
+          <SegmentedControl labels={controlLabels} onChange={onTabUpdate} selected={secureMessagingTab} a11yHints={a11yHints} />
```

Comment:
This needs the `a11yLabels` prop to send an override label for the screen reader for the "Inbox" label so it reads it without the count included (count is covered by the a11yHints). 

As a note: I made it so you only need to override the label you wanted in the index array... so sending `[t('secureMessaging.inbox')]` should be sufficient since "Folders" doesn't need any override since the text matches screen reader.